experimental/framework/fluid-static/src/containerCode.ts

```diff
@@ -130,6 +134,14 @@ export class RootDataObject
         return this._initialObjects;
     }
 
+    public get attachState() {
+        return this.context.attachState;
+    }
+
+    public get disposed() {
+        return super.disposed;
```

Comment:
I'm not confident these super calls are going to work the way we want (both here and in `dispose()`).  These refer to a data object's disposal, whereas we're trying to have this data object masquerade as a container.  Have you verified what these will do to the container?  These might need to reach to the container and make the calls there.

Reply:
yeah, taking a 2nd look, this isn't what we want. digging back in